CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+- Adds support for suffixes in SID and STAR procedures [#33](https://github.com/openscope/openscope/issues/33)
+- Adds game option to include/hide WIP airports in the airport list [#476](https://github.com/openscope/openscope/issues/476)
+- Adds `StripViewController`, `StripViewCollection`, and `StripViewModel` classes [#285](https://github.com/openscope/openscope/issues/285)
+    - Removes progress strip logic from the `AircraftModel`
+    - Completely reworks CSS for `StripViewList`
+- Adds `.isGroundedFlightPhase()` and implements this helper in `.buildWaypointModelsForProcedure()` [#491](https://github.com/openscope/openscope/issues/491)
+    - This allows for waypointModels to be build from the correct collection based on `flightPhase`
+- Updates `AircraftModel.onAirspaceExit()` to look only at the `mcp.headingMode` value [#477](https://github.com/openscope/openscope/issues/477)
+- Adds user setting option to change length of PTL [#423](https://github.com/openscope/openscope/issues/423)
+- Updates Dublin (EIDW) - Improved procedures, added terrain and video map, modified airspace, realistic traffic [#208](https://github.com/openscope/openscope/issues/208)
+- Updates logic to display historical aircraft position for aircraft outside controlled airspace [#508](https://github.com/openscope/openscope/issues/508)
+- Updates development-workflow-procedures, adds Quick Start guide to README and consolidates all documentation in the `documentation` directory [#418](https://github.com/openscope/openscope/issues/418)
+- Adds tests and verifies functionality of non-procedural departures and arrivals (support for direct route strings) [#434](https://github.com/openscope/openscope/issues/434)
+- Adds unique transponder and CID generation methods [#483](https://github.com/openscope/openscope/issues/483)
+- Abstracts non game loop logic into new class `AppController`, which provides facade methods for `App` to call during game loop [#137](https://github.com/openscope/openscope/issues/137)
+- Converts `AirportController`, `GameController` and `UiController` to static classes [#72](https://github.com/openscope/openscope/issues/72)
+    - updates `window` references to these classes
+    - updates `prop` references to these classes
+- Add support for ranged altitude and speed restrictions in procedures [#32](https://github.com/openscope/openscope/issues/32)
+- Improve VNAV climbs and descents to better comply with restrictions [#32](https://github.com/openscope/openscope/issues/32)
+- Updates Boston Logan Intl. (KBOS) - Updated procedures, added video map and terrain, modified airspace, realistic traffic [#228](https://github.com/openscope/openscope/issues/228)
+- Removes left over references to `AircraftStripView` in `AircraftModel` [#535](https://github.com/openscope/openscope/issues/535)
+
```

Comment:
we'll need to remove these extra spaces (its on the checklist, but just in case)